packages/config/package.json

```
@@ -4,7 +4,7 @@
   "description": "OpenCRVS public configuration microservice",
   "license": "MPL-2.0",
   "scripts": {
-    "start": "cross-env NODE_ENV=development nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
+    "start": "cross-env NODE_ENV=development NODE_OPTIONS=--dns-result-order=ipv4first  nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
```

```suggestion
    "start": "cross-env NODE_ENV=development NODE_OPTIONS=--dns-result-order=ipv4first nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
```
Very minor thing but some of these have double spaces in the command